scripts/nikonc2_curvefit_test.py

```diff
@@ -105,7 +105,7 @@ def main(args):
             # plt.show()
             try:
                 popt, pcov = curve_fit(arctan_func, td, s)
-            except RuntimeError as TypeError:
+            except RuntimeError as TypeError:  #TODO does that make sense?
```

Comment:
Nope, that doesn't make sense. It's from my "fix" for Python 3. The proper writting should be:
`except (RuntimeError, TypeError):`